.github/workflows/reusable-workflow.yml

```diff
@@ -0,0 +1,328 @@
+          working-directory: 'src'
+          list-tests: 'failed'
+
+#####################################################################
+# CLANG
+
+  clang:
+    runs-on: ubuntu-22.04
+    steps:
+
+      - name: Sync repository
+        uses: eProsima/eProsima-CI/external/checkout@v0
+        with:
+          path: ${{ github.workspace }}/src
+
+      - name: Download dependencies and install requirements
+        uses: ./src/.github/actions/project_dependencies
+        with:
+          os: ubuntu-22.04
+          cmake_build_type: Release
+          custom_version_build: ${{ inputs.custom_version_build }}
+          dependencies_artifact_postfix: ${{ inputs.dependencies_artifact_postfix }}
+          secret_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Compile and run tests
+        uses: eProsima/eProsima-CI/multiplatform/clang_build_test@v0
+        with:
+          packages_names: ${{ env.code_packages_names }}
+          workspace_dependencies: ${{ github.workspace }}/install
+
+
+#####################################################################
+# COVERAGE
+
+  coverage:
+    runs-on: ubuntu-22.04
+    environment:
+      name: codecov
+    steps:
+
+      - name: Sync repository
+        uses: eProsima/eProsima-CI/external/checkout@v0
+        with:
+          path: ${{ github.workspace }}/src
+
+      - name: Download dependencies and install requirements
+        uses: ./src/.github/actions/project_dependencies
+        with:
+          os: ubuntu-22.04
+          cmake_build_type: Release
+          custom_version_build: ${{ inputs.custom_version_build }}
+          dependencies_artifact_postfix: ${{ inputs.dependencies_artifact_postfix }}
+          secret_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Compile and run tests
+        uses: eProsima/eProsima-CI/ubuntu/coverage_build_test_upload@main
+        with:
+          packages_names: ${{ env.code_packages_names }}
+          workspace_dependencies: ${{ github.workspace }}/install
+          codecov_token: ${{ secrets.CODECOV_TOKEN }}
+          codecov_fix_file_path: ${{ github.workspace }}/src/codecov.yml
+
+
+#####################################################################
+# FLAKY
+
+  flaky:
+    runs-on: ubuntu-22.04
+    steps:
+
+      - name: Sync repository
+        uses: eProsima/eProsima-CI/external/checkout@v0
+        with:
+          path: ${{ github.workspace }}/src
+
+      - name: Download dependencies and install requirements
+        uses: ./src/.github/actions/project_dependencies
+        with:
+          os: ubuntu-22.04
+          cmake_build_type: Release
+          custom_version_build: ${{ inputs.custom_version_build }}
+          dependencies_artifact_postfix: ${{ inputs.dependencies_artifact_postfix }}
+          secret_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Compile and run tests
+        id: compile_and_test
+        uses: eProsima/eProsima-CI/multiplatform/colcon_build_test@main
```

Comment:
```suggestion
        uses: eProsima/eProsima-CI/multiplatform/colcon_build_test_flaky@main
```

Reply:
I'll remove colcon_build_test_flaky from eProsima-CI as soon as I update it on every repository as it is redundand and implies to maintain the same code twice